css/_welcome_page.scss

```diff
@@ -149,18 +150,22 @@ body.welcome-page {
     }
 
     .welcome-page-button {
-        width: $welcomePageButtonWidth;
-        min-width: $welcomePageButtonMinWidth;
-        height: $welcomePageButtonHeight;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        min-height: $welcomePageButtonMinHeight;
         font-size: $welcomePageButtonFontSize;
         font-weight: $welcomePageButtonFontWeight;
         background: #0074E0;
         border-radius: $welcomePageButtonBorderRadius;
         color: #FFFFFF;
-        text-align: center;
-        vertical-align: middle;
-        line-height: $welcomePageButtonLineHeight;
         cursor: pointer;
+        padding: 0 7px;
+
+        .welcome-page-button--text {
+            min-width: 33px; /* Keep the current width for "GO" */
```

Comment:
Please use `$welcomePageButtonMinWidth` here.

Reply:
I see, please wait for a follow-up. It will be up shortly.